src/Driver.cpp

```diff
@@ -343,6 +343,9 @@ static int createIPServerSocket(const char* port, addrinfo const& hints)
         if (sfd == -1)
             continue;
 
+        int option = 1;
+        setsockopt(sfd, SOL_SOCKET, SO_REUSEADDR, &option, sizeof(option));    
```

Comment:
Don't REUSEADDR by default.

You can use the URI options to enable it optionally if you'd like to.